src/librustc_typeck/check/demand.rs

```diff
@@ -374,6 +379,77 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
         None
     }
 
+    // This function checks if the specified expression is a built-in range literal
```

Comment:
Use `///` for documentation comments for functions. (Nit: And full stops for the ends of sentences 😄)

Reply:
Right. Does nightly rustdoc support code references yet?

Reply:
As in references to source files? Not that I'm aware of.